integration/controllers/carpool.ctrl.spec.ts

```diff
@@ -300,6 +314,42 @@ describe('Carpool controller', () => {
       .finally(done);
   });
 
+  it('should update a carpool', (done) => {
+    var carpool:models.Carpool;
+    var carpoolUpdate = {
+      'name': 'name updated',
+      'description': 'description updated'
+    };
+
+    function test(status) {
+      expect(status).toBe(204);
+
+      var updatedCarpool = CarpoolSvc.getCarpoolByID(carpool.id);
```

Comment:
It does *seem* to work. I was expecting it to cause problems but the test passes and it verifies that the values are correct. I will try to refactor it with a ``then()`` function